checklists/alz_checklist.en.json

```diff
@@ -2113,8 +2188,20 @@
             "guid": "aa45be6a-8f2d-4896-b0e3-885e6e94e770",
             "id": "F01.20",
             "severity": "Medium",
-            "link": "https://learn.microsoft.com/en-us/azure/azure-monitor/agents/azure-monitor-agent-overview",
-            "training": "https://learn.microsoft.com/en-us/azure/azure-monitor/agents/azure-monitor-agent-overview#installation"
+            "link": "https://learn.microsoft.com/azure/azure-monitor/agents/azure-monitor-agent-overview",
+            "training": "https://learn.microsoft.com/azure/azure-monitor/agents/azure-monitor-agent-overview#installation"
+        },
+        {
+            "category": "Management",
+            "subcategory": "Data Protection",
+            "text": "Ensure that storage accounts are zone or region redundant, Redundancy ensures storage accounts meet availability and durability targets amidst failures, weighing lower costs against higher availability. Locally redundant storage offers the least durability at the lowest cost.",
```

Comment:
Which storage accounts in the ALZ should be zone and region redundant?  Should this be one that we make as a "deploy a policy to manage redundancy" since that is part of the ALZ itself?  Or should we refer them to the storage checklist?  https://github.com/Azure/review-checklists/blob/main/checklists/azure_storage_checklist.en.json

Reply:
@brsteph all storage accounts should be reviewed and the ones which don't require ZRS or GRS need to be justified.

Reply:
But we don't deploy any storage accounts as part of the ALZ deployment - this looks like an application or workload centric review item more.

I'm fine with including it, but I want to make sure that we aren't bringing in workload items here.

What do you think of having the recommendation here be about the policy that requires the preferred sku?